apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3" [class]="customClass">
+    <ng-content select="[cardImage]"></ng-content>
+
+    <section>
+        <ng-container>
+            @for (item of list; track item; ) {
```

Comment:
you should prefer to target a property of item. item is a back default.

Reply:
I don't get it, Any example?

Reply:
item is quite useless. because since we use immutable data, the list will be rerender each time we delete or add an element. 
You should prefer $index as a default if we do not move/replace/sort element in the list, which we do. 
BUt the best is to target a property like item.id for instance

Reply:
Thanks!